pages/pipelines/command_step.md.erb

```diff
@@ -120,12 +120,51 @@ _Optional attributes:_
       <em>Example:</em> <code>"My reason"</code>
     </td>
   </tr>
+  <tr>
+    <td><code>retry</code></td>
+    <td>
+      Configuration of how retries (either <strong>automatic</strong> or <strong>manual</strong>) operate on this step<br>
+      Note: If automatic retries are enabled, a retry job will be triggered if it fails with any exit status <code>"*"</code> a maximum of 2 times.<br>
+
+      <br/>
+      <em>Example (automatic retries with defaults):</em><br>
+      <code>retry:</code><br>
+      <code>&nbsp;&nbsp;automatic: true</code><br/>
+
+      <br/>
+      <em>Example (different automatic retry rules depending on exit status):</em><br>
+      <code>retry:</code><br>
+      <code>&nbsp;&nbsp;automatic:</code><br/>
+      <code>&nbsp;&nbsp;&nbsp;&nbsp;- exit_status: 5</code><br/>
+      <code>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;limit: 2</code><br/>
+      <code>&nbsp;&nbsp;&nbsp;&nbsp;- exit_status: "*"</code><br/>
+      <code>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;limit: 4</code><br/>
+
+      <br/>
+      <em>Example (disabling manual retries):</em><br>
+      <code>retry:</code><br>
+      <code>&nbsp;&nbsp;manual: false</code><br>
+
+      <br/>
+      <em>Example (allowing retries on passed jobs):</em><br>
+      <code>retry:</code><br>
+      <code>&nbsp;&nbsp;manual:</code><br>
+      <code>&nbsp;&nbsp;&nbsp;&nbsp;permit_on_passed: true</code><br/>
```

Comment:
Are there any other options available under `manual`? Or is it just `permit_on_passed`?

Reply:
There's `allowed: true/false`, which you'll probably use along side the optional property `reason`

```
manual:
  allowed: false
  reason: "No retries allowed on deploy steps"
```

In the below example, the `reason` has no affect and will do nothing:

```
manual:
  allowed: true
  reason: "No retries allowed on deploy steps"
```

Also in this example, `permit_on_passed` will have no effect since we've turned off manual retries anyway.

```
manual:
  allowed: false
  permit_on_passed: true
```

Reply:
Is the `reason` field displayed somewhere in the UI?

Reply:
Yeah, the "Retry" button will be disabled in the UI with a ⚠ icon next to it. When you over over the button a tooltip while show you why it can't be retried (in this case, the user provided reason will be shown)

Reply:
![screen shot 2018-02-19 at 12 25 18 pm](https://user-images.githubusercontent.com/25882/36362459-fb715862-156f-11e8-9040-97833ba9ee92.png)

Reply:
One more question 😄does `permit_on_passed` have a default value?